.github/workflows/ci.yml

```diff
@@ -196,15 +196,30 @@ jobs:
             backend: cpp
             env: { MACOSX_DEPLOYMENT_TARGET: 10.14 }
 
+        exclude:
+          # fails github.com/cython/cython/issues/3450
+          - os: windows-2019
+            python-version: 3.9
```

Comment:
At the moment, I just do not run tests for these versions, but it would be better if I just ignore the problem areas. What's the best way to ignore a test for a specific python version on a specific platform?

Reply:
I really don't want to put this in windows_bugs.txt so that it will be ignored for all python versions.

Reply:
We can have `windows_bugs_38.txt` and `windows_bugs_39.txt`. If it's really resolved in 3.10, then let's do something simple.

Reply:
> windows_bugs_38.txt

Hmm is it disabled for py 3.8?

Reply:
> We can have `windows_bugs_38.txt` and `windows_bugs_39.txt`.

Seems like it'll be cool

Reply:
> If it's really resolved in 3.10, then let's do something simple.

Well, I will be testing inline additionally locally on version 3.10 (will the TestInline test case be enough?)

But I don’t understand, what simple can we do?

Reply:
> windows_bugs_38.txt

I thought it works with py3.8

Reply:
> I thought it works with py3.8

When launched locally, it really did not work, it probably works for ci, because it uses the newer version 3.8 (I have 3.8.1, and ci has 3.8.10)

Reply:
> We can have `windows_bugs_38.txt` and `windows_bugs_39.txt`.

Are there any new features we need to implement for this to work?  Or can we get it out of the box?

Reply:
> We can have `windows_bugs_38.txt` and `windows_bugs_39.txt`.

@scoder Are there any new features we need to implement for this to work?  Or can we get it out of the box?

Reply:
[partially resolved](https://github.com/cython/cython/pull/4324/commits/ffe34a394ba01f90ef8f7ea71243fd114f226204)

Reply:
[resolved](https://github.com/0dminnimda/cython/commit/e24b25ddf6867b3f71ae3d7b82b83067e1728ffe)